stream/street/import.js

```diff
@@ -17,78 +16,54 @@ function streamFactory(db, done){
   stats.tick();
 
   // create a new stream
-  return through.obj(function( batch, _, next ){
+  return through.obj(function (batch, _, next) {
```

Comment:
would be nice to make this not so deeply nested by pulling out a few functions into named variables, maybe processBatch and streamDone?